lib/cryptoexchange/exchanges/saturn_network/services/pairs.rb

```diff
@@ -0,0 +1,26 @@
+module Cryptoexchange::Exchanges
+  module SaturnNetwork
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::SaturnNetwork::Market::API_URL}/returnTicker.json"
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          output.map do |pair|
+            base = pair[0].split('_').first
+            target = pair[1]["symbol"]
```

Comment:
In our convention target is usually the dominant currency like USD, BTC, in this case ETH/ETC
base is the currency that varies